### PR TITLE
Minrelayfee func

### PR DESCRIPTION
**Pull request by gijswijs**
Added MinRelayFee func to walletkit_client.go.

#### Pull Request Checklist

- [x] PR is opened against correct version branch.
- [ ] Version compatibility matrix in the README and minimal required version
      in `lnd_services.go` are updated.
- [x] Update `macaroon_recipes.go` if your PR adds a new method that is called
  differently than the RPC method it invokes.

### CODE REVIEW

**Review by bhandras — APPROVED** [review]
LGTM 🌮

File: walletkit_client.go [hunk 1]
```diff
@@ -540,6 +540,25 @@ func (m *walletKitClient) EstimateFeeRate(ctx context.Context, confTarget int32)
 	return chainfee.SatPerKWeight(resp.SatPerKw), nil
 }
 
+// MinRelayFee returns the current minimum relay fee based on our chain backend
+// in sat/kw.
+func (m *walletKitClient) MinRelayFee(ctx context.Context) (
```
Comment (guggero) [inline walletkit_client.go:545]: nit: I would've formatted this as:

```
func (m *walletKitClient) MinRelayFee(
	ctx context.Context) (chainfee.SatPerKWeight, error) {
```

But non-blocking.

**Review by guggero — APPROVED** [review]
LGTM :tada: 

I though we could do this against the `master` branch, but it turns out the new field was actually only added in `0.18.x`, so `lnd-18-4` is the correct branch.

**Comment by guggero** [comment]
Can you squash the commits? Then I think we can merge this. Or is anything else missing here? Just waiting to confirm everything works in `tapd`?

**Comment by gijswijs** [comment]
> Just waiting to confirm everything works in `tapd`?
Exactly. All seems to work. So we have a cascade of PRs depending on this one.

**Pull request merged**